app/assets/javascripts/psp.js

```diff
@@ -3,7 +3,7 @@ GOVUKFrontend.initAll();
 if (document.querySelector('#country') != null) {
     openregisterLocationPicker({
         selectElement: document.getElementById('country'),
-        url: '/register-as-pension-scheme-practitioner/assets/javascripts/autocomplete/location-autocomplete-graph.json'
+        url: '/pension-scheme-practitioner/register/assets/javascripts/autocomplete/location-autocomplete-graph.json'
```

Comment:
Is there any reason why the assets are not at the root level?

Reply:
Assets are accessed using the base URL via routes as in: 
`GET /assets/*file controllers.Assets.versioned(path="/public", file: Asset)` 
so this path is correct from a routing perspective even though it looks wrong from a package structure perspective.

Reply:
addressed comment